### ISSUE TITLE
Dice loss

### ISSUE THREAD

**Issue by yiqian-wang** [Jiaxuan-Li/MGU-Net#3]
In the Dice loss, there's a line:
input = torch.exp(input) 
but shouldn't it be the sigmoid function? The input is the last conv layer output from the network and a sigmoid will convert it to probability within [0,1] 
input = torch.sigmoid(input)

**Comment by yiqian-wang** [comment]
*The input is after log_softmax()
